<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PGrantManDockWidgetBase</class>
 <widget class="QDockWidget" name="PGrantManDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1604</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PGrantMan</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tw_grp_roles">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Login Rollen</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Add Login Role</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tw_loginroles">
          <property name="rowCount">
           <number>0</number>
          </property>
          <property name="columnCount">
           <number>3</number>
          </property>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Kommentar</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Mitglied von</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Gruppen</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Add Group Role</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeWidget" name="tree_grp_roles">
          <attribute name="headerDefaultSectionSize">
           <number>1000</number>
          </attribute>
          <column>
           <property name="text">
            <string>Group Roles</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Privilegien</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>PG Verbindung</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cb_pgconns">
            <property name="sizeAdjustPolicy">
             <enum>QComboBox::AdjustToContents</enum>
            </property>
            <item>
             <property name="text">
              <string>PG Verbindung auswählen ...</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_dontlistsystables">
            <property name="text">
             <string>Systemtabellen ausblenden (schneller!)</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidget_2">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_4">
           <attribute name="title">
            <string>Tabellen</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QTableWidget" name="tw_tables">
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <attribute name="horizontalHeaderCascadingSectionResizes">
               <bool>true</bool>
              </attribute>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>false</bool>
              </attribute>
              <column>
               <property name="text">
                <string>Schema.Tabelle</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Typ</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>ALL</string>
               </property>
               <property name="textAlignment">
                <set>AlignHCenter|AlignVCenter|AlignCenter</set>
               </property>
              </column>
              <column>
               <property name="text">
                <string>INSERT</string>
               </property>
               <property name="textAlignment">
                <set>AlignHCenter|AlignVCenter|AlignCenter</set>
               </property>
              </column>
              <column>
               <property name="text">
                <string>SELECT</string>
               </property>
               <property name="textAlignment">
                <set>AlignHCenter|AlignVCenter|AlignCenter</set>
               </property>
              </column>
              <column>
               <property name="text">
                <string>UPDATE</string>
               </property>
               <property name="textAlignment">
                <set>AlignHCenter|AlignVCenter|AlignCenter</set>
               </property>
              </column>
              <column>
               <property name="text">
                <string>DELETE</string>
               </property>
               <property name="textAlignment">
                <set>AlignHCenter|AlignVCenter|AlignCenter</set>
               </property>
              </column>
              <column>
               <property name="text">
                <string>TRUNCATE</string>
               </property>
               <property name="textAlignment">
                <set>AlignHCenter|AlignVCenter|AlignCenter</set>
               </property>
              </column>
              <column>
               <property name="text">
                <string>REF</string>
               </property>
               <property name="textAlignment">
                <set>AlignHCenter|AlignVCenter|AlignCenter</set>
               </property>
              </column>
              <column>
               <property name="text">
                <string>TRIGGER</string>
               </property>
               <property name="textAlignment">
                <set>AlignHCenter|AlignVCenter|AlignCenter</set>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_4">
                <property name="text">
                 <string>Rückgängig</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_3">
                <property name="text">
                 <string>Übernehmen</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_5">
           <attribute name="title">
            <string>Funktionen</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
